css/index.css

```diff
@@ -1 +1,380 @@
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  /* 1 rem === 10px based on the user setting of "medium" which is 16px */
+  font-size: 62.5%;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! |||||||||||||||||||||||||||||||||||||||||| */
+
+.container {
+	max-width: 880px;
+    margin: 0 auto;
+    padding: 20px;
+}
+
+p {
+    font-size: 1.5rem;
+    line-height: 1.25;
+    font-weight: bold;
+}
+
+h2 {
+    font-size: 2rem;
+}
+
+header {
+	display: flex;
+	justify-content: center;
+	height: 80px;
+	width: 100%;
+	background: white;
+	position: fixed;
+	top: 0;
+	left: 0;
+	
+}
+
+.wrapper {
+	display: flex;
+	align-items: center;
+	justify-content: space-between;
+	/* width: 877px; */
+}
+
+nav {
+    display: flex;
+    justify-content: space-between;
+    width: 600px;
+
+}
+nav a {
+	text-decoration: none;
+	color: lightgrey;
+	font-size: 1.6rem;
+	width: 16%;
+
+}
+
+.logo {
+	display: flex;
+    justify-content: flex-end;
+    max-width: 180px;
+    width: 100%;
+    
+}
+
+.hero {
+	margin: 125px 0 75px;
+}
+
+.title {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	text-align: center;
+	margin-top: 65px;
+	
+}
+
+.title h1 {
+	font-size: 8rem;
+}
+
+.title a {
+	border: 1px solid black;
+	padding: 0px 30px;
+	text-decoration: none;
+	color: black;
+	font-size: 2rem;
+
+}
+
+/* main section /////////////////*/
+main {
+	border-top: 1px solid black;
+	border-bottom: 1px solid black;
+}
+
+main, .middle-img {
+    width: 100%;
+	padding: 40px 0;
+
+}
+
+.half div, .thirds div {
+	white-space: normal;
+	display: inline-flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	/* height: 100%; */
+	vertical-align: top;
+}
+
+.half {
+	white-space: nowrap;
+	
+}
+
+.half div   {
+	width: 50%;
+}
+
+.thirds {
+    display: flex;
+    justify-content: space-between;
+}
+
+.thirds div {
+    width: 32%;
+}
+
+
+.row {
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 40px;
+}
+
+/* footer //////////////////////// */
+footer {
+	margin-top: 40px;
+}
+
+footer a {
+	display: flex;
+	text-decoration: none;
+	color: black;
+}
+
+.phone {
+	margin: 24px 0;
+	
+}
+
+#copyright {
+	text-align: center;
+	margin: 48px 0 108px;
+	font-size: 14px;
+}
+
+/* //////////////////////////////// */
+@media all and (max-width: 800px) {
+   .container {
+       width: 100%;
+       padding: 20px;
+   }
+
+   header {
+       margin-top: 32px;
```

Comment:
You may have noticed that you can see content above your fixed nav when you scroll down. Remember that background color only applies to elements inside of the border.